README.md

```diff
@@ -500,6 +500,7 @@ Options:
                                                                                                    # Default: .
              [--halt-upon-load-error], [--no-halt-upon-load-error], [--skip-halt-upon-load-error]  # Halt upon a load error while loading the Rails application
                                                                                                    # Default: true
+             [--skip-constant=constant [constant ...]]                                             # Skip the given application constant(s) when generating RBIs
```

Comment:
While the name of the option is okay, I think we should use something else than `skip` in the description to better explain what it is doing:

```suggestion
             [--skip-constant=constant [constant ...]]                                             # Do not generate RBI definition for the given application constant(s)
```